recipes/pulpy/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "pulpy" %}
+{% set version = "1.8.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 3036611410c6a1213d20ae7512cc00444d402d761f31f593e506144b1e0a3a74
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation"
+
+requirements:
+  host:
+    - pip
+    - python >=3.9
+    - setuptools
+    - numpy <=1.20
+    - pywavelets
+    - numba
+    - scipy <=1.10.0
+    - tqdm
+    - sigpy >=0.1.26
+  run:
```

Comment:
I'm not sure about these requirements as they don't appear in the `pyproject.toml` or `setup.py`/`setup.cfg`, but I'm assuming they are ok.

Reply:
Some were trimmed from the initial project and are now removed both here and in pyproject.toml